amp/WEB-INF/src/org/digijava/module/aim/util/AuditLoggerUtil.java

```diff
@@ -259,6 +259,14 @@ public static void logActivityUpdate(HttpServletRequest request, AmpActivityVers
         }
     }
     
+    public static List<String> getUsersFromLog() {
```

Comment:
Since we have a constraint in audit logger table for user id we should the a list of users, so if the name changes we can see the change. Its different with teams that we store only the team name and not a reference to the team id